src/elements/hx-search/_shadow.scss

```diff
@@ -17,10 +17,9 @@ input::-ms-clear {
     -ms-grid-column-align: start;
     -ms-grid-columns: 2rem 1fr;
     -ms-grid-rows: 1fr;
-    display: -ms-grid;
-
     align-items: center;
-    display: grid;
+    display: -ms-grid; // Legacy (IE)
+    display: grid; // Modern
```

Comment:
rearranged to satisfy [`declaration-block-no-duplicate-properties`](https://stylelint.io/user-guide/rules/declaration-block-no-duplicate-properties) configuration (ignoring consecutive props with different values)